### DISCUSSION TITLE
Access to real time data

### DISCUSSION THREAD

**Discussion by Viktor1722** [br-g/openf1#46] [Q&A] [answered]
Hello i found this API to be perfect for a university project which is going to be a app providing real time data about the race. Currently i am trying to use the sessions endpoint and for some reason i cant see the (for example) dates and time of the next Belgium gp i can only see it for 2023 race and i have tried to make it work but without success. Can it be access related because i found in a reddit post that the plan is to make the real time data payed. I am building the project with Next.js 14

**Comment by br-g** [comment]
Hello,
Only past data is available currently

**Reply by gabrielebellini01** [reply]
Hi @br-g, when will real-time data be available again during races?
Thank you for your time and effort!

**Reply by br-g** [reply, marked as answer]
Hello,
Data is already available in real-time during races.
What is not yet available is data about the future (planning of future GPs)

**Reply by gabrielebellini01** [reply]
Thank you for your fast reply! Will future sessions data ever be available?

**Reply by br-g** [reply]
Yes it will be available eventually!
The current focus, however, is data quality, cost efficiency, and open-source collaboration

**Reply by Viktor1722** [reply]
Thank you for the response
